tests/ord-aggregator/tests/handler_test.go

```diff
@@ -398,3 +411,20 @@ func storeMappingBetweenORDAndInternalBundleID(t *testing.T, respBody string, nu
 
 	return ordAndInternalIDsMapping
 }
+
+func getGlobalResourcesNumber(ctx context.Context, httpClient *http.Client) (int, int) {
+	certCache, err := certloader.StartCertLoader(ctx, testConfig.CertLoaderConfig)
+	if err != nil {
+		log.C(ctx).Errorf("Failed to initialize certificate loader %v", err)
+		return 0, 0
+	}
+
+	accessStrategyExecutorProvider := accessstrategy.NewDefaultExecutorProvider(certCache)
+	ordClient := NewClient(httpClient, accessStrategyExecutorProvider)
+
+	products, vendors, err := ordClient.GetGlobalProductsAndVendorsNumber(ctx, testConfig.GlobalRegistryURL)
+	if err != nil {
+		log.C(ctx).Errorf("while fetching global registry resources from %s %v", testConfig.GlobalRegistryURL, err)
+	}
```

Comment:
Pass testing.T in this function and fail the test on error with a meaningful message. Otherwise, it will be hard to figure out why the numbers differ in the asserts.